src/pages/home/index.js

```diff
@@ -4,6 +4,7 @@ import { FormattedMessage } from 'react-intl'
 import Hero from 'shared/components/hero-section'
 import Features from 'shared/components/features-section'
 import GettingStarted from 'shared/components/getting-started-section'
+import ServiceWorkerSection from 'shared/components/service-worker-section'
```

Comment:
I think that this section should be called `GatewaySection` because it uses a gateway for getting `IPFS` files. Moreover, this gateway can be a service worker gateway, or the `js-ipfs` daemon gateway.